smurfs/src/components/AddSmurfForm.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux';
+import { createSmurf } from '../actions';
+
+class AddSmurfForm extends Component {
+
+    state = {
+        smurfs: {
+            name: '',
+            age: '',
```

Comment:
I think `smurfs` is a little confusing, I would probably go with `smurf` or even something even more descriptive